lib/galaxy/workflow/modules.py

```diff
@@ -1777,7 +1769,7 @@ def from_workflow_step(Class, trans, step, **kwds):
         module.post_job_actions = {}
         for pja in step.post_job_actions:
             module.post_job_actions[pja.action_type] = pja
-        if module.tool:
+        if module.tool and module.version_changes:
```

Comment:
Not sure I understand this change?

Reply:
eh, that was not supposed to go in there, thanks for the catch